components/TheMenu.vue

```diff
@@ -128,5 +112,37 @@ export default {
       mobileMenu.classList.toggle("block");
     },
   },
+  data: function () {
+    return {
+      menuLinks: [
+        {
+          name: "What We Do",
+          href: "/#what-we-do",
+          isDesktop: true,
+          isMobile: true,
+        },
+        {
+          name: "Get Involved",
+          href: "/#get-involved",
+          isDesktop: true,
+          isMobile: true,
+        },
+        {
+          name: "Contact",
+          href: "/contact",
+          isDesktop: true,
+          isMobile: true,
+        },
+      ],
+    };
+  },
+  computed: {
+    mobileLinks: function () {
+      return this.menuLinks.filter((link) => link.isMobile);
+    },
```

Comment:
Looks great @dboydgit. Can I ask why you're using `isDesktop` and `isMobile` to filter the menu items if they're both always true? Seems like an interesting feature but also one we could add later if needed.

Reply:
@MarkBennett I've updated this branch again to keep it up to date, took out the mobile vs desktop menus, we can add later if there's a use case. Also fixed up some invalid html in the board_recruiting page since that was throwing some errors.